package.json

```diff
@@ -115,5 +115,8 @@
     "svgo": "0.6.1",
     "webpack": "1.12.12",
     "webpack-dev-server": "1.14.1"
+  },
+  "dependencies": {
+    "eidolon": "^1.2.0"
```

Comment:
Yes you can, using the [server build](https://github.com/apiaryio/attributes-kit/blob/master/package.json#L34)